lib/frontier/controller_action/index_action.rb

```diff
@@ -6,14 +6,37 @@ def to_s
     raw = <<-STRING
 def index
   #{Frontier::Authorization::Assertion.new(model_configuration, :index).to_s}
-  #{model_configuration.as_ivar_collection} = #{scopable_object}.page(params[:page])
+#{Frontier::RubyRenderer.new(object_scope).render(1)}
```

Comment:
it's intentional. The render method will format all lines at a given line. See the specs for more info